ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+**DO NOT ask questions using GitHub issues (only bug or feature requests please see http://loopback.io/doc/en/contrib/Reporting-issues.html#asking-questions)**
+**Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs or https://gitter.im/strongloop/loopback**
+**Immediate support is available through our subscription plans, see https://strongloop.com/node-js/subscription-plans**
```

Comment:
Let's use markdown comments for this. That way users filling a new issue will see the message, but the rendered issue won't show it.

```markdown
<!--
DO NOT ask questions using GitHub issues (only bug or feature requests please see http://loopback.io/doc/en/contrib/Reporting-issues.html#asking-questions)**
Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs or https://gitter.im/strongloop/loopback**
Immediate support is available through our subscription plans, see https://strongloop.com/node-js/subscription-plans
-->
```